enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemDisplay.unit.spec.tsx

```diff
@@ -153,10 +186,50 @@ describe("SdkUsageProblemDisplay", () => {
 
     await userEvent.click(screen.getByTestId(PROBLEM_INDICATOR_TEST_ID));
 
+    const card = screen.getByTestId(PROBLEM_CARD_TEST_ID);
+
     expect(
-      within(screen.getByTestId(PROBLEM_CARD_TEST_ID)).getByText(
+      within(card).getByText(
         /cannot use both an Auth Provider URI and API key authentication at the same time/,
       ),
     ).toBeInTheDocument();
+
+    const docsLink = within(card).getByRole("link", {
+      name: /View documentation/,
+    });
+
+    expect(docsLink).toHaveAttribute(
+      "href",
+      "https://www.metabase.com/docs/latest/embedding/sdk/authentication#authenticating-people-from-your-server",
+    );
+  });
+
+  // Caveat: we cannot detect this on non-localhost environments, as
+  // CORS is disabled on /api/session/properties.
+  it("shows an error when Embedding SDK is disabled on localhost", async () => {
+    setup({
+      authConfig: createMockAuthProviderUriConfig(),
+      hasEmbeddingFeature: true,
+      isEmbeddingSdkEnabled: false,
+    });
+
+    await userEvent.click(screen.getByTestId(PROBLEM_INDICATOR_TEST_ID));
+
+    const card = screen.getByTestId(PROBLEM_CARD_TEST_ID);
+
+    expect(
+      within(card).getByText(
+        /The embedding SDK is not enabled for this instance. Please enable it in settings to start using the SDK./,
+      ),
+    ).toBeInTheDocument();
+
+    const docsLink = within(card).getByRole("link", {
+      name: /View documentation/,
+    });
+
+    expect(docsLink).toHaveAttribute(
+      "href",
+      "https://www.metabase.com/docs/latest/embedding/sdk/introduction#in-metabase",
+    );
```

Comment:
Show the SDK introduction docs if they have not enabled SDK embedding yet. Read the below comment to see why this detection only works in localhost.